src/utils/license-utilities.js

```diff
@@ -30,7 +30,7 @@ const shortToSlug = {
  * @param short
  * @returns {string}
  */
-const slugFromShort = short => {
+const slugFromShort = (short) => {
```

Comment:
These changes might be an artifact of VSCode autoformatting.  I can revert if desired.

Reply:
let's revert the autoformatting if you can, we try to do that in separate Issues/PRs.